scout/server/app.py

```diff
@@ -100,6 +101,7 @@ def register_blueprints(app):
     app.register_blueprint(api.api_bp)
     app.register_blueprint(alignviewers.alignviewers_bp)
     app.register_blueprint(phenotypes.hpo_bp)
+    app.register_blueprint(institutes.institutesov_bp)
```

Comment:
Don't know how final the name of this page is, but I don't like much 'institutesov'. Stands for institute overview right? what about just overview?